### PR TITLE
Update translator acknowledgements

### PR DESCRIPTION
**Pull request by lukassim**
## Description

Update the Translation Program and translation acknowledgement pages to replace the mentions of POAPs with OATs and update the process to claim them

## Related Issue

No related issue

### CODE REVIEW

**Review by minimalsm — APPROVED** [review]

File: src/pages/contributing/translation-program/acknowledgements.tsx [hunk 1]
```diff
@@ -249,14 +249,14 @@ const TranslatorAcknowledgements = () => {
...
-      <Content id="poap">
+      <Content id="oats">
         <ContentHeading
           as="h2"
           fontSize={{ base: "2xl", md: "2rem" }}
           fontWeight={600}
         >
           {t(
-            "page-contributing-translation-program-acknowledgements-poaps-title"
+            "page-contributing-translation-program-acknowledgements-oats-title"
```
Comment (wackerow) [inline src/pages/contributing/translation-program/acknowledgements.tsx:259]: @lukassim 

image

We end up with a weird case here... The above .json string changes are in the English files only... we changed this string name, but this doesn't exist in other languages yet, so it falls back to the English "OATs"... but the other strings do exists in those langs and have the old copy.

I'd suggest we update this string key in all languages as well, which will display POAP along with everything else in there until these get translated.

Comment (lukassim) [reply src/pages/contributing/translation-program/acknowledgements.tsx:259]: Good point, though I'd argue OAT shouldn't be translated anyway, same as with POAP
But yeah, there's still mention of POAP in all the subsequent text...

Comment (lukassim) [reply src/pages/contributing/translation-program/acknowledgements.tsx:259]: One thing to note is that this page doesn't get that many views even in English, and even less in other languages, so I'd also be fine with keeping it this way until the new copy gets translated

Comment (wackerow) [reply src/pages/contributing/translation-program/acknowledgements.tsx:259]: Hah okay... I just went ahead and fixed that key so the non-English pages will just be consistently not yet updated... the updates will then trickle down in cohesive chunks at least... and like you said, not a lot of eyes.

**Review by wackerow — APPROVED** [review]

**Pull request merged**